test/parallel/test-diagnostics-channel-tracing-channel-sync.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common');
+const dc = require('diagnostics_channel');
+const assert = require('assert');
+
+const channel = dc.tracingChannel('test');
+
+const expectedResult = { foo: 'bar' };
+const input = { foo: 'bar' };
+
+function check(found) {
+  assert.deepStrictEqual(found, input);
```

Comment:
```suggestion
  assert.strictEqual(found, input);
```

similar in other tests